docker-compose.yml

```diff
@@ -1,53 +1,83 @@
 version: '2'
 services:
 
-  postgres:
-    image: postgres
+  database:
+    image: camptocamp/postgis:9.5
```

Comment:
The official install guide has postgis as a dependency (http://docs.geonode.org/en/master/tutorials/admin/install/custom_install.html) - is this different?